ocaml/Makefile.common-jst

```diff
@@ -60,7 +60,8 @@ boot_targets = \
   $(boot_ocamlopt) \
   $(boot_ocamlmklib) \
   $(boot_ocamldep) \
-  $(boot_ocamlobjinfo)
+  $(boot_ocamlobjinfo) \
+  ocaml/ocamltest/ocamltest.byte
```

Comment:
(Is it a left-over?)

Reply:
Ah no, I forgot to describe this change.